src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -921,7 +921,7 @@ public static void Main()
             .VerifyEmitDiagnostics(new CodeAnalysis.Emit.EmitOptions(runtimeMetadataVersion: "x.y"),
                 // (15,21): error CS0117: 'string' does not contain a definition for 'Format'
```

Comment:
yep, another test is being added for this case as well with an alignment to check for that diagnostic. I wasn't exactly sure what to do here because this test is marked `[WorkItem(1097428, "DevDiv")]`. I could make this test the one that looks for `Format` and a second that looks for `Concat` or I could make them the other way. Either way I need to change this test. should I add the attribute to the new test as well?